### DISCUSSION TITLE
An option to add Genymotion to the desktop entry

### DISCUSSION THREAD

**Discussion by AndnixSH** [Genymobile/feedback#51] [Genymotion Desktop] [answered]
It would be nice if you can add an option to add Genymotion to the desktop entry for easier starting it from start menu on Linux system, just like what Android Studio does via Tools > Create Desktop Entry

![image](https://user-images.githubusercontent.com/40742924/232252584-ef178f81-850a-41d5-a9b6-4d27ca16d5e1.png)

**Comment by jmaneyrol69** [comment]
Hi @AndnixSH ,
Could you give us more details? For example, which distribution and desktop do you use (Gnome, KDE, xfce, etc.).
Thank you.

**Reply by AndnixSH** [reply]
I'm using Linux Mint 21, Cinnamon Desktop based on Ubuntu

**Reply by jmaneyrol69** [reply, marked as answer]
Thank you for the details.
Just to let you know: we do not support Liunx Mint 21 nor Cinnamon Desktop. If we add this feature in the future, it will only be supported on Ubuntu, Debian and Fedora Workstation with GNOME, KDE, XFCE and MATE desktop.

Thank you for your understanding.

**Reply by AndnixSH** [reply]
That's unfortunate but thanks!
